src/main/java/com/bts/personalbudget/core/domain/service/fixedbill/FixedBillService.java

```diff
@@ -1,13 +1,194 @@
+        log.info("m=save fixedBill={}", fixedBill);
+        validationMandatoryFields(fixedBill);
+        validationDays(fixedBill);
+        fixedBill.setStatus(FixedBillStatus.ACTIVE);
+        final FixedBillEntity fixedBillEntity = buildFixedBillEntity(fixedBill);
+        final List<CalendarFixedBillEntity> calendarFixedBillEntityList = buildCalendarFixedBillEntityList(fixedBill, fixedBillEntity);
+        fixedBillEntity.setCalendarFixedBillEntityList(calendarFixedBillEntityList);
+        fixedBillRepository.save(fixedBillEntity);
+    }
+
+    private List<CalendarFixedBillEntity> buildCalendarFixedBillEntityList(FixedBill fixedBill, FixedBillEntity fixedBillEntity) {
+        log.info("m=buildCalendarFixedBillEntityList fixedBillDays={} fixedBillCode={}", fixedBill.getDays(), fixedBillEntity.getCode());
+        final List<CalendarFixedBillEntity> calendarFixedBillEntityList = new ArrayList<>();
+
+        for (Integer day : fixedBill.getDays()) {
+            CalendarFixedBillEntity calendarFixedBillEntity = new CalendarFixedBillEntity();
+            calendarFixedBillEntity.setDayLaunch(day);
+            calendarFixedBillEntity.setFlgLeapYear(fixedBill.getFlgLeapYear());
+            calendarFixedBillEntity.setFlgActive(Boolean.TRUE);
+            calendarFixedBillEntity.setFixedBill(fixedBillEntity);
+            calendarFixedBillEntityList.add(calendarFixedBillEntity);
+        }
+        return calendarFixedBillEntityList;
+    }
+
+    private FixedBillEntity buildFixedBillEntity(final FixedBill fixedBill) {
+        final FixedBillEntity fixedBillEntity = fixedBillMapper.toEntity(fixedBill);
+        fixedBillEntity.setCode(UUID.randomUUID());
+        fixedBillEntity.setFlagActive(true);
+        fixedBillEntity.setStatus(FixedBillStatus.ACTIVE);
+        final LocalDate paramDate = defineNextDueDate(fixedBill, LocalDate.now()).orElse(null);
+        fixedBillEntity.setNextDueDate(paramDate);
+        return fixedBillEntity;
+    }
+
+    public FixedBill findByCode(final UUID code) {
+        final FixedBillEntity fixedBillEntity = findEntity(code);
+        return fixedBillMapper.toModel(fixedBillEntity);
+    }
+
+    public List<FixedBill> findAll() {
+        final List<FixedBillEntity> fixedBillEntityList = fixedBillRepository.findAllByFlagActiveTrue();
+        return fixedBillMapper.toModelList(fixedBillEntityList);
+    }
+
+    public FixedBill update(final UUID code,
+                            final OperationType operationType,
+                            final String description,
+                            final BigDecimal amount,
+                            final RecurrenceType recurrenceType,
+                            final List<Integer> days,
+                            final Boolean flgLeapYear,
+                            final FixedBillStatus status,
+                            final LocalDate startDate,
+                            final LocalDate endDate) {
```

Comment:
add logs no método update

```suggestion
                            final LocalDate endDate) {
                            log.info("m=update, code={}, operationType={}... ", code, operationType, ....)

```